build-tests/api-extractor-lib2-test/etc/api-extractor-lib2-test.api.md

```diff
@@ -7,7 +7,6 @@
 // @public (undocumented)
 class DefaultClass {
 }
-
```

Comment:
I'm not sure why this whitespace is changing here and in some other files, but I believe these changes are wanted. 🤷  It makes API report files more consistent.

Reply:
Not sure. I ran `common\autoinstallers\rush-prettier\node_modules\.bin\prettier -w .` in the repo root folder for your branch, and it didn't produce any diff.